index.html

```diff
@@ -30,34 +30,34 @@ <h1 class="head">I’m Suzana <br> Glad to see you!</h1>
         <p class="pr">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
         <h6 class="a">LET’S CONNECT</h6>
         <ul class="social-media-container">
-          <li class="i"><a href="#"><img src="images/angellist.png" alt=""></a></li>
-          <li class="i"><a href="#"><img src="images/twitter.png" alt=""></a></li>
-          <li class="i"><a href="#"><img src="images/github.png" alt=""></a></li>
-          <li class="i"><a href="#"><img src="images/medium.png" alt=""></a></li>
-          <li class="i"><a href="#"><img src="images/Linkedinicon.png" alt=""></a></li>
+          <li class="i"><a href="#"><img src="images/angellist.png" alt=" icons"></a></li>
+          <li class="i"><a href="#"><img src="images/twitter.png" alt="icons"></a></li>
+          <li class="i"><a href="#"><img src="images/github.png" alt="icons"></a></li>
+          <li class="i"><a href="#"><img src="images/medium.png" alt="icons"></a></li>
+          <li class="i"><a href="#"><img src="images/Linkedinicon.png" alt="icons"></a></li>
```

Comment:
- [OPTIONAL] You should have added a link to this changes in the report summary  as specified in the requirements, checkout this [link](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148) on how to go about it.